hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/hudi/functional/TestColumnStatsIndex.scala

```diff
@@ -84,10 +85,11 @@ class TestColumnStatsIndex extends HoodieClientTestBase {
       HoodieMetadataConfig.ENABLE_METADATA_INDEX_COLUMN_STATS.key -> "true"
     )
 
-    val opts = Map(
+    val hudiOpts = Map(
```

Comment:
All of the opts are Hudi opts, maybe `commonOpts`?

Reply:
Sounds good.  Fixed.